src/django_components/extension.py

```diff
@@ -27,26 +27,38 @@
 ################################################
 
 
+# Mark a class as an extension hook context so we can place these in
+# a separate documentation section
+def mark_extension_hook_api(cls: Type[Any]) -> Type[Any]:
```

Comment:
To be able to mark which objects should be considered as related to extension hooks / urls / commands, I added little decorators like these.